actionview/lib/action_view/routing_url_for.rb

```diff
@@ -1,3 +1,6 @@
+gem 'actionpack', ActionView::VERSION::STRING
+require 'action_pack'
+
 require 'action_dispatch/routing/polymorphic_routes'
```

Comment:
This is a really bad coupling. The only way to remove it is to define a simple version of this method in Action View and enhance it when Action Pack is present.